docs/feature_mouse_keys.md

```diff
@@ -21,30 +22,36 @@ You can see an example here: https://github.com/qmk/qmk_firmware/blob/master/key
 
 You can use these keycodes within your keymap to map button presses to mouse actions:
 
-|Key             |Aliases  |Description                |
-|----------------|---------|---------------------------|
-|`KC_MS_UP`      |`KC_MS_U`|Mouse Cursor Up            |
-|`KC_MS_DOWN`    |`KC_MS_D`|Mouse Cursor Down          |
-|`KC_MS_LEFT`    |`KC_MS_L`|Mouse Cursor Left          |
-|`KC_MS_RIGHT`   |`KC_MS_R`|Mouse Cursor Right         |
-|`KC_MS_BTN1`    |`KC_BTN1`|Mouse Button 1             |
-|`KC_MS_BTN2`    |`KC_BTN2`|Mouse Button 2             |
-|`KC_MS_BTN3`    |`KC_BTN3`|Mouse Button 3             |
-|`KC_MS_BTN4`    |`KC_BTN4`|Mouse Button 4             |
-|`KC_MS_BTN5`    |`KC_BTN5`|Mouse Button 5             |
-|`KC_MS_WH_UP`   |`KC_WH_U`|Mouse Wheel Up             |
-|`KC_MS_WH_DOWN` |`KC_WH_D`|Mouse Wheel Down           |
-|`KC_MS_WH_LEFT` |`KC_WH_L`|Mouse Wheel Left           |
-|`KC_MS_WH_RIGHT`|`KC_WH_R`|Mouse Wheel Right          |
-|`KC_MS_ACCEL0`  |`KC_ACL0`|Set mouse acceleration to 0|
-|`KC_MS_ACCEL1`  |`KC_ACL1`|Set mouse acceleration to 1|
-|`KC_MS_ACCEL2`  |`KC_ACL2`|Set mouse acceleration to 2|
+|Key             |Aliases  |Description                         |
+|----------------|---------|------------------------------------|
+|`KC_MS_UP`      |`KC_MS_U`|Mouse Cursor Up                     |
+|`KC_MS_DOWN`    |`KC_MS_D`|Mouse Cursor Down                   |
+|`KC_MS_LEFT`    |`KC_MS_L`|Mouse Cursor Left                   |
+|`KC_MS_RIGHT`   |`KC_MS_R`|Mouse Cursor Right                  |
+|`KC_MS_BTN1`    |`KC_BTN1`|Mouse Button 1                      |
+|`KC_MS_BTN2`    |`KC_BTN2`|Mouse Button 2                      |
+|`KC_MS_BTN3`    |`KC_BTN3`|Mouse Button 3                      |
+|`KC_MS_BTN4`    |`KC_BTN4`|Mouse Button 4                      |
+|`KC_MS_BTN5`    |`KC_BTN5`|Mouse Button 5                      |
+|`KC_MS_WH_UP`   |`KC_WH_U`|Mouse Wheel Up                      |
+|`KC_MS_WH_DOWN` |`KC_WH_D`|Mouse Wheel Down                    |
+|`KC_MS_WH_LEFT` |`KC_WH_L`|Mouse Wheel Left                    |
+|`KC_MS_WH_RIGHT`|`KC_WH_R`|Mouse Wheel Right                   |
+|`KC_MS_ACCEL0`  |`KC_ACL0`|Set mouse acceleration to 0 (slow)  |
+|`KC_MS_ACCEL1`  |`KC_ACL1`|Set mouse acceleration to 1 (medium)|
+|`KC_MS_ACCEL2`  |`KC_ACL2`|Set mouse acceleration to 2 (fast)  |
 
 You can see an example in the `_ML` here: https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/66/keymaps/mouse_keys/keymap.c#L46
 
 ## Configuring the Behavior of Mousekeys
 
-The default speed for controlling the mouse with the keyboard is intentionally slow. You can adjust these parameters by adding these settings to your keymap's `config.h` file. All times are specified in milliseconds (ms).
+Regardless of whether you use cursor acceleration or constant speeds, all times are specified in milliseconds (ms).
```

Comment:
Could you add some introductory text before this? Something like, "There are two movement models you can select between for mousekeys. Cursor acceleration [short description]. By contrast, Constant Cursor [short description]."